spec/cheets_spec.rb

```diff
@@ -0,0 +1,18 @@
+require './lib/cheets'
+
+describe Cheets do
+
+  it { should have_property   :id         }
```

Comment:
As we discussed in the group session, you probably don't need to be testing these things. When we're writing declarative code without any behaviour there's nothing really there for a unit test; and the model itself should be covered by a feature test so you still have the coverage.